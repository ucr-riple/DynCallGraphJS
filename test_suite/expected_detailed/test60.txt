{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:1:1:12:3)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:1:2:8:2)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:7:10:7:16)': [
        'foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:2:3:6:4)'
    ],
    'foo (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:3:5:5:7)': [
        'call (Native)(/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:3:5:5:7)'
    ],
    'call (Native) (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:3:5:5:7)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:9:3:11:4)'
    ],
    'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:10:5:10:12)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test60.js:3:27:5:6)'
    ]
}