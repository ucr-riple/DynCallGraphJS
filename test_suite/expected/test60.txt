{ '(test60.js:1:1:12:3)':
   [ '(test60.js:1:2:8:2)' ],
  '(test60.js:7:10:7:16)':
   [ '(test60.js:2:3:6:4)' ],
  '(test60.js:3:5:5:7)': [ 'call (Native)' ],
  'call (Native)':
   [ '(test60.js:9:3:11:4)' ],
  '(test60.js:10:5:10:12)':
   [ '(test60.js:3:27:5:6)' ] }